src/gravel/controllers/bootstrap.py

```diff
@@ -77,6 +77,15 @@ async def bootstrap(self) -> bool:
 
         return True
 
+    async def mark_finished(self) -> bool:
+        stage: DeploymentStage = gstate.config.deployment_state.stage
+        if stage == DeploymentStage.ready:
+            return True
+        elif stage != DeploymentStage.bootstrapped:
+            return False
+        gstate.config.set_deployment_stage(DeploymentStage.ready)
```

Comment:
is it really necessary to have the web client set the ready flag? ... maybe it should be set at the end of a successful bootstrap?

Reply:
and then we have no idea if we have already went through the whole installer. :shrug:

Reply:
Maybe we need to separate the concept of deployment_stage from "wizard/installer stage"?